### ISSUE TITLE
Robert Wilton DISCUSS (2)

### ISSUE THREAD

**Issue by vixie** [ietf-wg-dnsop/draft-ietf-dnsop-avoid-fragmentation#6]
(2) p 3, sec 3.2.  Recommendations for UDP requestors

   R6.  UDP requestors SHOULD limit the requestor's maximum UDP payload
   size to the RECOMMENDED size of 1400 or a smaller size.

I find this recommendation to be unclear because it mixes both a "SHOULD" and
"RECOMMENDED", i.e., I find it unclear as to what the "SHOULD" applies to.  Is
the recommendation (i) that UDP requestors should limit the maximum UDP
payload.  Or (ii) is the recommendation that a limit of 1400 be used, or (iii)
perhaps both.  Maybe rewording this to something like the following would help:

   R6.  UDP requestors SHOULD limit the requestor's maximum UDP payload
   size to 1400 bytes, but MAY limit the maximum UDP payload size to a
   smaller size on small MTU (less than 1500 bytes) networks.

   or,

   R6.  UDP requestors SHOULD limit the requestor's maximum UDP payload
   size.  It is RECOMMENDED to use a limit of 1400 bytes, but a smaller
   limit MAY be used.

**Comment by paulwouters** [comment]
I thought it was clear but can't see why not to use your second suggestion, so did that in my proposed PR to please everyone.
